lib/railway_ipc.rb

```diff
@@ -1,26 +1,27 @@
-require "railway_ipc/version"
-require "sneakers"
-require "bunny"
-require "railway_ipc/version"
-require "railway_ipc/logger"
-require "railway_ipc/unhandled_message_error"
-require "railway_ipc/response"
-require "railway_ipc/rabbitmq/payload"
-require "railway_ipc/null_message"
-require "railway_ipc/concerns/message_handling"
-require "railway_ipc/rabbitmq/connection"
-require "railway_ipc/handler"
-require "railway_ipc/consumer"
-require "railway_ipc/publisher"
-require "railway_ipc/null_handler"
-require "railway_ipc/responder"
-require "railway_ipc/client"
-require "railway_ipc/server"
-require "railway_ipc/railtie" if defined?(Rails)
+require 'railway_ipc/version'
```

Comment:
Tool changed a lot of these to single quotes. New Classes worth remarking about will be called out.